src/main/features/claimant-response/form/models/rejectionReason.ts

```diff
@@ -0,0 +1,32 @@
+import { IsDefined, MaxLength } from 'class-validator'
+import { IsNotBlank } from '@hmcts/cmc-validators'
+import { ValidationConstraints } from 'forms/validation/validationConstraints'
+import { ValidationErrors as DefaultValidationErrors } from 'forms/validation/validationErrors'
+
+export class ValidationErrors {
+  static readonly EXPLAIN_WHY_YOU_REJECT_REPAYMENT_PLAN = 'Enter an explanation of why you reject repayment plan'
```

Comment:
is this from content?
It doesn't really make sense to me
@RonanFitzgerald @cidlockie

Reply:
There is no mention of validation for this page, i have put that as a placeholder. I will update it once i speak to content guys